test/api/mailboxes-test.js

```diff
@@ -146,88 +146,315 @@ describe('Mailboxes tests', function () {
     });
 
     it('should GET /users/{user}/mailboxes expect success / all params', async () => {
-        const response = await server.get(`/users/${user}/mailboxes`).send({ specialUse: true, showHidden: true, counters: true, sizes: true }).expect(200);
+        const response = await server.get(`/users/${user}/mailboxes?specialUse=true&showHidden=true&counters=true&sizes=true`).send({}).expect(200);
```

Comment:
Here and in other places - you don't need `send()` for GET requests. You can shorten it to `server.get(...).expect(200)`

Reply:
Fixed